dbt_integration.py

```diff
@@ -59,6 +61,17 @@
     from dbt.adapters.base import BaseRelation  # type: ignore
     from dbt.contracts.connection import AdapterResponse
 
+import logging
+
+# Set up logging to file
+logging.basicConfig(filename="python_log.log", level=logging.DEBUG)
+logger = logging.getLogger(__name__)
```

Comment:
Are we going to introduce the logger here?

I think we shouldn't log by default.

Reply:
No this is only until i finish debugging. this should be gone when its ready. keeping the comment as a todo

Reply:
this is removed